src/shared/icons/block.svg

```diff
@@ -1 +1,10 @@
-<svg width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="6" stroke="#7B8A95" stroke-width="2" stroke-linecap="round"/><path d="M4 4l8 8" stroke="currentColor" stroke-width="2" stroke-linecap="round"/></svg>
\ No newline at end of file
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Not sure if you're aware of this tool: https://jakearchibald.github.io/svgomg/

It's a really great way to clean up SVGs and reduce their size. I usually copy the full SVG content and paste it into "Paste markup". Then I tick the `prettify` option and copy markup. In this case the result is only 40% size of the original

Let's put this icon through it